eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -156,42 +156,35 @@ class NormalStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     val initialState = alice.stateData.asInstanceOf[DATA_NORMAL]
     val add = CMD_ADD_HTLC(MilliSatoshi(Int.MaxValue), randomBytes32, CltvExpiryDelta(144).toCltvExpiry, TestConstants.emptyOnionPacket, upstream = Left(UUID.randomUUID()))
     sender.send(alice, add)
-    val error = InsufficientFunds(channelId(alice), amount = MilliSatoshi(Int.MaxValue), missing = Satoshi(1376443), reserve = Satoshi(20000), fees = Satoshi(8960), isLocal = true)
+    val error = InsufficientFunds(channelId(alice), amount = MilliSatoshi(Int.MaxValue), missing = 1376443 sat, reserve = 20000 sat, fees = 8960 sat)
     sender.expectMsg(Failure(AddHtlcFailed(channelId(alice), add.paymentHash, error, Local(add.upstream.left.get, Some(sender.ref)), Some(initialState.channelUpdate), Some(add))))
     alice2bob.expectNoMsg(200 millis)
   }
 
-  test("recv CMD_ADD_HTLC (HTLC dips remote funder below reserve)") { f =>
+  test("recv CMD_ADD_HTLC (insufficient funds, missing 1 msat)") { f =>
     import f._
     val sender = TestProbe()
-    val (_, htlc) = addHtlc(MilliSatoshi(771000000), alice, bob, alice2bob, bob2alice)
-
-    sender.send(alice, CMD_SIGN)
-    sender.expectMsg("ok")
-    alice2bob.expectMsgType[CommitSig]
-    alice2bob.forward(bob)
-    bob2alice.expectMsgType[RevokeAndAck]
-    bob2alice.forward(alice)
-    awaitCond(alice.stateData.asInstanceOf[DATA_NORMAL].commitments.remoteNextCommitInfo.isRight)
-
-    bob2alice.expectMsgType[CommitSig]
-    bob2alice.forward(alice)
+    val initialState = bob.stateData.asInstanceOf[DATA_NORMAL]
+    val add = CMD_ADD_HTLC(initialState.commitments.availableBalanceForSend + 1.msat, randomBytes32, CltvExpiryDelta(144).toCltvExpiry, TestConstants.emptyOnionPacket, upstream = Left(UUID.randomUUID()))
+    sender.send(bob, add)
 
-    // at this point bob still hasn't forwarded the htlc downstream
-    relayerB.expectNoMsg()
+    val error = InsufficientFunds(channelId(alice), amount = add.amount, missing = 1 sat, reserve = 10000 sat, fees = 8960 sat)
```

Comment:
Changes in `sendAdd` affect this test, I've updated values and `missing` is `1 sat` instead of `0`, not sure if this is correct or maybe defeats a purpose of test.